code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -274,14 +274,6 @@
 		M.visible_message("<span class='notice'>[M] hugs [src] to make [p_them()] feel better!</span>", \
 					"<span class='notice'>You hug [src] to make [p_them()] feel better!</span>")
 
-		// Warm them up with hugs
-		share_bodytemperature(M)
```

Comment:
Why are you deleting a proccall thats not a moodlet? 

share_bodytemperature is a carbon proc